build.gradle

```diff
@@ -1,8 +1,15 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2024.3.2"
+    id "edu.wpi.first.GradleRIO" version "2025.1.1-beta-3"
     id("com.diffplug.spotless") version "7.0.0.BETA2"
```

Comment:
Can we just bump to latest spotless? also this should be changed to be consistent with the other plugins